### PR TITLE
Update whoami.c

### PR DESCRIPTION
**Pull request by yungyuantseng**

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("1022988");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by jrjang2** [comment]
Merge all commits into one before doing pull request.

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by yungyuantseng** [comment]
done!!

**Comment by jrjang2** [comment]
Good!

**Pull request merged**